feature-libs/product-configurator/rulebased/components/group-title/configurator-group-title.component.ts

```diff
@@ -80,4 +125,13 @@ export class ConfiguratorGroupTitleComponent {
     }
     return title;
   }
+
+  /**
+   * Verifies whether the current screen size equals or is smaller than breakpoint `BREAKPOINT.md`.
+   *
+   * @returns {Observable<boolean>} - If the given breakpoint equals or is smaller than`BREAKPOINT.md` returns `true`, otherwise `false`.
+   */
+  isMobile(): Observable<boolean> | undefined {
```

Comment:
Shouldn't we return of(false) in case breakpoint service is not present?

Reply:
Yes! Agree! Adjusted!